### DISCUSSION TITLE
Can't build from repository

### DISCUSSION THREAD

**Discussion by youngkaneda** [herbstluftwm/herbstluftwm#1503] [Q&A] [answered]
I tried to build the wm because my version from debian package is outdated, my problem is that I have all the dependencies needed, but it still fail on the cmake step because I do not have an "xfixes" package, I could only find a xcb-xfixes dev package but the error remains, appreciate any help!

**Comment by The-Compiler** [comment, marked as answer]
Not much to say with so little details... What's the exact error and what distribution are you on?

**Reply by youngkaneda** [reply]
My problem is that I was missing a package, that you wasn't finding on my searchs, libfixes-dev, one person on hlwm irc chat pointed me this package, and worked!
